simplq/src/components/pages/Home/Nav/LeftNav.jsx

```diff
@@ -12,24 +12,26 @@ const scrollToHowItWorks = () => {
 
 const LeftNav = ({ open }) => {
   return (
-    <ul className={styles['left-nav']} open={open}>
-      <li>
-        <a tabIndex={0} role="link" onKeyDown={scrollToHowItWorks} onClick={scrollToHowItWorks}>
-          How it works
-        </a>
-      </li>
-      <li>
-        <a>My Queues</a>
-      </li>
-      <li>
-        <a tabIndex={-1} href="https://iimb.qualtrics.com/jfe/form/SV_aY8FY91ztRX9NvD">
-          Contact Us
-        </a>
-      </li>
-      <li>
-        <a>Sign In/Sign Up</a>
-      </li>
-    </ul>
+    <div>
+      <ul className={styles['left-nav']} open={open}>
+        <li>
+          <a tabIndex={0} role="link" onKeyDown={scrollToHowItWorks} onClick={scrollToHowItWorks}>
+            How it works
+          </a>
+        </li>
+        <li>
+          <a>My Queues</a>
+        </li>
+        <li>
+          <a tabIndex={-1} href="https://iimb.qualtrics.com/jfe/form/SV_aY8FY91ztRX9NvD">
+            Contact Us
+          </a>
+        </li>
+        <li>
+          <a>Sign In/Sign Up</a>
+        </li>
```

Comment:
Is there any change here? Or was it just removing the left in the scss file that fixed this?

Reply:
Also, wrapped the entire <ul> under a <div> tag in LeftNav.jsx component to get it working :)

Reply:
+1 How did this fix the isssue?

Reply:
There are multiple ways to fix this. Ideally, it is something to do with the way CSS positioning has been used. Wrapping in a div, would ensure that anything around the div is consuming its entire width and height, giving space for other div's as well. In this case, the `<ul>` was not wrapped in a div and the browser couldn't calculate exact width consumed by it, hence the left property pulled it on top of the logo text :)
I know it sounds confusing, but hope it helps.

Reply:
Okay, I didn't know that. Thanks.